extensions/reactive-messaging-http/deployment/pom.xml

```diff
@@ -73,9 +73,9 @@
             <plugin>
                 <artifactId>maven-surefire-plugin</artifactId>
                 <configuration>
-                    <systemProperties>
+                    <systemPropertyVariables>
                         <java.util.logging.manager>org.jboss.logmanager.LogManager</java.util.logging.manager>
```

Comment:
This is actually entirely superfluous, see `build-parent`.
But I guess this PR should be merged as is and I'll come up with another PR to drop those unecessary surefire configs from `extensions/*`.

Reply:
thanks!